src/components/experiment/ExperimentItem/__tests__/ExperimentItem.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExperimentItem from '../ExperimentItem';
+import { act } from 'react-dom/test-utils';
+import { Modal } from 'react-bootstrap';
+
+describe('<ExperimentItem />', () => {
+  it('renders without crashing', () => {
+    shallow(
+      <ExperimentItem url="http://localhost/static/experiment/some_experiment_url.py" />
+    );
+  });
+
+  it('renders filename text parsed from URL', () => {
+    const wrapper = shallow(
+      <ExperimentItem url="http://localhost/static/experiment/some_experiment_url.py" />
+    );
+    expect(wrapper.contains('some_experiment_url.py')).toEqual(true);
+  });
+
+  it('renders a model when the delete button is clicked.', async () => {
+    const wrapper = shallow(
+      <ExperimentItem url="http://localhost/static/experiment/some_experiment_url.py" />
+    );
+    await act(async () => {
+      wrapper.find('.delete-button').simulate('click');
+    });
+    expect(wrapper.find(Modal).props()['show']).toBe(true);
+  });
+
+  it('Clicking delete and then delete again issues a fetch', async () => {
```

Comment:
Nitpick: this should be phrased more like:
it('issues a deletion call when delete and delete confirmation are clicked")

Reply:
makes sense